src/CopyButton.js

```diff
@@ -20,7 +20,7 @@ export function useCopyTextHandler(textToCopy, { interval } = {}) {
 }
 
 export const CopyButton = React.memo(function CopyButton(props) {
-  return props.children(useCopyTextHandler(props.text, { interval: props.interval }))
+  return props.children(useCopyTextHandler(props.text, props.options))
```

Comment:
`props.interval` must remain, otherwise this would be a breaking change. 

Can we just do:

```suggestion
  return props.children(useCopyTextHandler(props.text, props))
```
??